### PR TITLE
ODH release 1.0 manifest

### PR DESCRIPTION
**Pull request by LaVLaS**
Adds the v1.0 release OLM manifest that will be submitted to OperatorHub for community operators

### CODE REVIEW

File: deploy/olm-catalog/opendatahub/1.0.0/manifests/opendatahub-operator.v1.0.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+    * Airflow v1.10.11 - Workflow management
+    * Seldon v1.2.0 - Open source platform for deploying machine learning models
+    * Argo v2.12.5 - Container-native Workflow Engine
+    * Apache Superset v0.30.0 - Open source application for data exploration and visualization
+    * Apache Kafka v2.4.0 - The open source stream processing platform
+    To install one or multiple of these components use the default kfDef provided with the operator.
+
+    ### Kubeflow Components
+    ODH 1.0 supports Kubeflow v1.2.0 and some components such as KF Serving and KFP on Tekton from master branch. This will be updated when Kueflow v1.3.0 is releases. To install Kubeflow v1.2.0 components with KFP using argo please use this example KfDef (https://github.com/kubeflow/manifests/blob/master/kfdef/kfctl_openshift.v1.2.0.yaml). To install KF Serving with KFP on Tekton please use this example KfDef (https://raw.githubusercontent.com/kubeflow/manifests/master/kfdef/kfctl_openshift.master.kfptekton.yaml)
+
```
Comment (nakfour) [inline deploy/olm-catalog/opendatahub/1.0.0/manifests/opendatahub-operator.v1.0.0.clusterserviceversion.yaml:249]: Typo "is released"

File: deploy/olm-catalog/opendatahub/1.0.0/manifests/opendatahub-operator.v1.0.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -222,27 +222,30 @@ spec:
       displayName: Open Data Hub
       group: kfdef.apps.kubeflow.org
   description: |
-    The Open Data Hub is a machine-learning-as-a-service platform built on Red Hat's Kubernetes-based OpenShift® Container Platform, Ceph Object Storage, and integrating a collection of open source projects.
+    The Open Data Hub is a machine-learning-as-a-service platform built on Red Hat's Kubernetes-based OpenShift® Container Platform. Open Data Hub integrates multiple AI/ML open source components into one operator that can easily be downloaded and installed by Openshift users.
```
Comment (anishasthana) [inline deploy/olm-catalog/opendatahub/1.0.0/manifests/opendatahub-operator.v1.0.0.clusterserviceversion.yaml:225]: NIT: Openshift -> OpenShift

**Review by anishasthana — CHANGES_REQUESTED** [review]

File: deploy/olm-catalog/opendatahub/1.0.0/manifests/opendatahub-operator.v1.0.0.clusterserviceversion.yaml (hunk 1 above)
Comment (nakfour) [reply deploy/olm-catalog/opendatahub/1.0.0/manifests/opendatahub-operator.v1.0.0.clusterserviceversion.yaml:249]: @LaVLaS one more fix, please use the raw url for kfdef "https://raw.githubusercontent.com/kubeflow/manifests/master/kfdef/kfctl_openshift.v1.2.0.yaml"

**Review by nakfour — APPROVED** [review]

File: deploy/olm-catalog/opendatahub/1.0.0/manifests/opendatahub-operator.v1.0.0.clusterserviceversion.yaml (hunk 2 above)
Comment (LaVLaS) [reply deploy/olm-catalog/opendatahub/1.0.0/manifests/opendatahub-operator.v1.0.0.clusterserviceversion.yaml]: Good catch

**Review by nakfour — APPROVED** [review]

**Review by anishasthana — APPROVED** [review]
/lgtm

**Review by nakfour — APPROVED** [review]

**Pull request merged**